locales/hi/translation.json

```
@@ -0,0 +1,259 @@
+        "remove-entry": "प्रविष्टि निकालें",
+        "switch-view": "दृश्य स्विच करें",
+        "time-to-leave": "Time to Leave",
+        "waived-day": "माफ किया हुआ दिन",
+        "yes": "हाँ"
+    },
+    "$FlexibleMonthCalendar": {
+        "add-waiver-day": "इस दिन के लिए एक छूट जोड़ें",
+        "current-month": "वर्तमान माह पर जाएं",
+        "day": "दिन",
+        "day-done-balance": "सब आज के लिए किया। दिन का संतुलन:",
+        "entry-removal-confirmation": "क्या आप वाकई दिन से अंतिम दो प्रविष्टियाँ निकालना चाहते हैं",
+        "last-day-balance": "अंतिम दिन का उपयोग संतुलन के लिए किया जाता है",
+        "leave-by": "आज आप जिस समय पर आए हैं, उसके आधार पर आपको छोड़ देना चाहिए",
+        "month-balance": "महीना शेष",
+        "month-balance-title": "इस महीने के लिए आज तक शेष राशि। एक सकारात्मक संतुलन का मतलब है अतिरिक्त घंटे जो आपको आज (या बाकी महीने) काम करने की आवश्यकता नहीं है।",
+        "next-month": "अगले महीने",
+        "no": "नहीं",
+        "on": "पर",
+        "overall-balance": "समग्र संतुलन",
+        "overall-balance-title": "महीने के अंत या वर्तमान दिन तक कुल मिलाकर शेष",
+        "previous-month": "पिछ्ला महिना",
+        "remove-entry": "प्रविष्टि निकालें",
+        "switch-view": "दृश्य स्विच करें",
+        "time-to-leave": "Time to Leave",
+        "total": "कुल",
+        "waived-day": "माफ किया हुआ दिन",
+        "working-days": "कार्य दिवस",
+        "working-days-title": "महीने में कितने कार्य दिवस होते हैं",
+        "yes": "हाँ"
+    },
+    "$Calendar": {
+        "all-done": "सब आज के लिए किया। दिन का संतुलन:",
+        "current-month": "वर्तमान माह",
+        "current-month-title": "वर्तमान माह पर जाएं",
+        "day": "दिन",
+        "day-end": "डे एंड",
+        "day-start": "दिन की शुरुआत",
+        "day-total": "दिन का कुल",
+        "last-day-balance": "अंतिम दिन का उपयोग संतुलन के लिए किया जाता है",
+        "leave-by": "आज आप जिस समय पर आए हैं, उसके आधार पर आपको छोड़ देना चाहिए",
+        "lunch-begin": "दोपहर का भोजन शुरू",
+        "lunch-end": "दोपहर का भोजन",
+        "lunch-start": "दोपहर का खाना शुरू",
+        "lunch-total": "दोपहर का भोजन कुल",
+        "month-balance": "महीना शेष",
+        "month-balance-title": "इस महीने के लिए आज तक शेष राशि। एक सकारात्मक संतुलन का मतलब है अतिरिक्त घंटे जो आपको आज (या महीने के बाकी) काम करने की आवश्यकता नहीं है।",
+        "next-month": "अगले महीने",
+        "on": "पर",
+        "overall-balance": "समग्र संतुलन",
+        "overall-balance-title": "महीने के अंत या वर्तमान दिन तक कुल मिलाकर शेष",
+        "previous-month": "पिछ्ला महिना",
+        "punch-time": "पंच समय",
+        "switch-view": "दृश्य स्विच करें",
+        "switch-view-title": "दृश्य स्विच करें",
+        "time-to-leave": "Time To Leave",
+        "working-days": "कार्य दिवस",
+        "working-days-title": "महीने में कितने कार्य दिवस होते हैं"
+    },
+    "$Notification": {
+        "punch-reminder": "में पंच करने के लिए मत भूलना!",
+        "time-to-leave": "Time to Leave"
```

I think this one was correct before. The base string for this is `Hey there! I think it's time to leave.`. I think it was:

```suggestion
        "time-to-leave": "सुनो! मुझे लगता है कि यह छोड़ने का समय है।"
```